src/session/sessions.ts

```diff
@@ -3,6 +3,10 @@
 //#INSTALL "graphql"
 // Contains the interface for gql queries, mutations and subscriptions
 import { DocumentNode } from "graphql";
+//> Cookie Utils
```

Comment:
Please remove this comment.

Reply:
I see no reason the remove it. I will replace it with one that fits.
In the upcoming documentation update all imports of the client are refined to fit the seek-tonic anyways.

Ref: https://github.com/snek-at/tonic